File: Python Exams/Basic Exam Collection/Basics Online Retake Exam - 2 and 3 May 2019/02. Safari/02. Safari.py
budget = float(input())
fuel = int(input())
day_of_week = input()

fuel_price = 2.10
gid_price = 100

subtotal = fuel * fuel_price + gid_price

if day_of_week == 'Saturday':
    subtotal *= 0.90
elif day_of_week == 'Sunday':
    subtotal *= 0.80


total = abs(budget - subtotal)

if budget >= subtotal :
    print(f'Safari time! Money left: {total:.2f} lv.')
else:
    print(f'Not enough money! Money needed: {total:.2f} lv.')



'''
120
30
Saturday
'''

File: JS Exams/01.  Basics Exam Collection/Basics Online Retake Exam - 2 and 3 May 2019/02. Safari/02. Safari.js
function safari(budget, fuel, dayOfWeek) {

    let fuelPrice = 2.10
    let gidPrice = 100
    let subtotal = fuel * fuelPrice + gidPrice
    
    dayOfWeek == 'Saturday' ? subtotal *= 0.90 : subtotal *= 0.80

    let total = Math.abs(budget - subtotal)

    if (budget >= subtotal)
        return `Safari time! Money left: ${total.toFixed(2)} lv.`
    return `Not enough money! Money needed: ${total.toFixed(2)} lv.`
}


//console.log(safari(120, 30, 'Saturday'))